src/extension.ts

```diff
@@ -78,7 +54,7 @@ export function activate(context: vscode.ExtensionContext) {
 		if (!args.webview) {
 
 			frameworkSelection =
-				await vscodeUtils.showQuickDialog(frameworkFacade.getFrameworks(defaultFramework as constants.Frameworks), constants.SELECT_FRAMEWORK, false);
+				await vscodeUtils.showQuickDialog(frameworkFacade.getFrameworks(defaultFramework as constants.Frameworks[]), constants.SELECT_FRAMEWORK, false);
```

Comment:
`getFrameworksFromWorkspace` should contains the return type instead of using type casting.